### PR TITLE
Javascript javascript1 week4

### PR DESCRIPTION
**Pull request by melo4yxa77796**
I still not good with git it is show some different my home work

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,309 +1,100 @@
...
-  let title = "";
-  let fullname = "";
-
-  if (useFormalName) {
-      if (gender.toLowerCase() === "male") {
-          title = "Lord ";
-      } else {
-          title = "Lady ";
-      }
+  if (command.startsWith("add ")) {
```
Comment (vdaguenet) [inline index.js:34]: It looks like you already implemented this command a few lines above.

File: index.js [hunk 2]
```diff
@@ -1,309 +1,100 @@
...
+  if (command === "what is on my todo") {
+    if (todos.length > 0) {
+      return `You have ${todos.length} todos: ${todos.join(", ")}.`;
+    } else {
+      return "You have no todos.";
     }
+  }
+  if (command === "what day is it today") {
+    const today = new Date();
+    return `Today is ${today}`;
```
Comment (vdaguenet) [inline index.js:58]: Do you know a way to display the date in a nicer format?

File: index.js [hunk 3]
```diff
@@ -1,309 +1,100 @@
...
-    totalPercentage += percentageOfLife;
-    console.log(`${series.title} took ${percentageOfLife.toFixed(3)}% of my life`);
-  });
-
-  console.log(`In total that is ${totalPercentage.toFixed(3)}% of my life`);
-}
-
-logOutSeriesText();
+function getReply(command) {
+  command = command.toLowerCase();
```
Comment (vdaguenet) [inline index.js:11]: Good idea for using `toLowerCase`! Another good practice with user inputs is to `trim` them too.

**Review by vdaguenet — APPROVED** [review]
Good job! I only wrote minor comments if you want to go further.

**Comment by vdaguenet** [comment]
> I still not good with git it is show some different my home work

If you want a cleaner PR diff, you can create different files for each week.

**Comment by melo4yxa77796** [comment]
thanks for comments

**Pull request merged**